src/dao/pages/DAOProposals/CreateDAOProposal.tsx

```diff
@@ -30,16 +30,22 @@ import {
   useCreateMultiSigDAOUpgradeProposal,
   useDAOs,
   useCreateGOVTokenAssociateProposal,
+  useContract,
 } from "@dao/hooks";
 import { useHandleTransactionSuccess, useAccountTokenBalances } from "@dex/hooks";
 import { isNil } from "ramda";
 import { TransactionResponse } from "@hashgraph/sdk";
 import { getLastPathInRoute } from "@dex/utils";
 import { getDAOType, getPreviousMemberAddress } from "../utils";
-import { solidityAddressToAccountIdString } from "@shared/utils";
+import { isNFT, solidityAddressToAccountIdString } from "@shared/utils";
 
 export function CreateDAOProposal() {
   const { accountId: daoAccountId = "" } = useParams();
+  const daoAccountIdQueryResults = useContract(daoAccountId);
+  const daoAccountEVMAddress = daoAccountIdQueryResults.data?.data.evm_address;
+  const daosQueryResults = useDAOs();
+  const { data: daos } = daosQueryResults;
+  const dao = daos?.find((dao) => dao.accountId.toLowerCase() == daoAccountEVMAddress?.toLowerCase());
```

Comment:
can we pass `daoEvmAddress` in hook and in `select` callback filter the desired dao and then return to component again.? it will save lines in lots of components.

Reply:
There are 2 problems with this.
1. useDAOs hook caches the data, based on the params. If we pass the evm address to hook, it will call the same API again for all the DAOs. 
2. useDAOS hook return data in array format as the same hook is being used for DAOs list page too, so we would again need to filter in component level anyway.